src/functions-and-arrays.js

```diff
@@ -1,73 +1,500 @@
-// Iteration #1: Find the maximum
+function maxOfTwoNumbers(n1, n2) {
+  if (n1 > n2) {
+    return n1;
+  } else if (n2 > n1) {
+    return n2;
+  } else {
+    return n1;
```

Comment:
You could have saved and else if like this: 
if (n1 > n2) {
    return n1;
  } else{return n2}